session/session.go

```diff
@@ -2025,15 +2025,14 @@ func (s *session) PrepareTxnCtx(ctx context.Context) {
 	}
 }
 
-// PrepareTSFuture uses to try to get txn future.
+// PrepareTSFuture uses to try to get ts future.
 func (s *session) PrepareTSFuture(ctx context.Context) {
 	if !s.txn.validOrPending() {
+		// Prepare the transaction future if the transaction is invalid (at the beginning of the transaction).
 		txnFuture := s.getTxnFuture(ctx)
 		s.txn.changeInvalidToPending(txnFuture)
-		s.GetSessionVars().TxnCtx.SetStmtFuture(txnFuture.future)
-		return
-	}
-	if s.txn.Valid() && s.GetSessionVars().IsPessimisticReadConsistency() {
+	} else if s.txn.Valid() && s.GetSessionVars().IsPessimisticReadConsistency() {
+		// Prepare the statement future if the transaction is valid in RC transactions.
 		s.GetSessionVars().TxnCtx.SetStmtFuture(s.getTxnFuture(ctx).future)
```

Comment:
The `getTxnFuture` and `SetStmtFuture` are only used for `RC`， renaming  them to `getTxnFutureForRC` like format to avoid using mistakenly?